docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,159 @@
+
+- [PMM-7806](https://jira.percona.com/browse/PMM-7806): Upgraded postgres_exporter version used in pmm from 0.8.0 to 0.10.1. With this upgrade, you can access the latest features with all the bug fixes in place. 
+
+  
+- [PMM-10018](https://jira.percona.com/browse/PMM-10018): Lowered the default values of CPU and memory for haproxy while deploying a DBaaS [Percona XtraDB Cluster](https://www.percona.com/software/mysql-database/percona-xtradb-cluster) (PXC) cluster to avoid wasting resources.
+
+
+    
+- [PMM-9118](https://jira.percona.com/browse/PMM-9118): Refined the error message thrown when a secure connection cannot be established while adding a monitoring database to PMM, thus making it much more contextual and easy to troubleshoot.
+
+
+- [PMM-10516](https://jira.percona.com/browse/PMM-10516): 
+
+- [PMM-10095](https://jira.percona.com/browse/PMM-10095): While removing the service with pmm-admin, now you do not have to specify the service name for a single service. PMM will automatically look up this service and remove it.
+
+- [PMM-9989](https://jira.percona.com/browse/PMM-9989): PMM now supports Percona Operator for MongoDB 1.12.0.
+
+
+
+## Bugs Fixed
+
+- [PMM-10628](https://jira.percona.com/browse/PMM-10628): While adding MongoDB or PostgreSQL database via socket, PMM was throwing an error. This has been fixed in PMM 2.31.0.
+
+- [PMM-10624](https://jira.percona.com/browse/PMM-10624): Empty dataname label after PostgreSQL upgraded in PMM.
+
+- [PMM-10608](https://jira.percona.com/browse/PMM-10608): Fixed issues pertaining to the new Home Dashboard, such as high CPU panel unreadable for a higher number of nodes (more than ten nodes), filtering for the environment as well as node name not working correctly, time range not working as expected, some panels on the dashboard not displaying data for selected filters, and so on.
+
+- [PMM-10587](https://jira.percona.com/browse/PMM-10587): Fixed an issue where if you hover over the telemetry icon stacktrace with error appears.
+
+- [PMM-10513](https://jira.percona.com/browse/PMM-10513): Fixed an issue where pmm-admin summary stored PMM password in client/status.json, thus posing a security risk.
+
+- [PMM-10065](https://jira.percona.com/browse/PMM-10065): Fixed an issue encountered while monitoring a large number of nodes (1000) and services. Here, the changing scrape frequency led to data not being captured on the dashboard and throwing a templating error.
+
+
+- [PMM-9973](https://jira.percona.com/browse/PMM-9973): Fixed an issue where the CPU utilization for Postgres Exporter was on the higher side.
+
+- [PMM-9891 ()]https://jira.percona.com/browse/PMM-9891): Fixed an issue where QAN truncated long queries.
+
+- [PMM-9556](https://jira.percona.com/browse/PMM-9556): Fixed an issue where creating a database cluster for MongoDB 5.0x was failing in DBaaS.
```

Comment:
This wasn't fixed. @percona-csalguero we probably need to remove this and add Alert/Note that ppl shouldn't use Mongo 5 with operators 1.10, 1.11 ?

Reply:
ok